### PR TITLE
Rider session: CE custom operations completion list

### PR DESCRIPTION
**Pull request by edgarfgp**
Hi @jindraivanek. This PR adds a session page with
-  Proposed date  and time  for https://github.com/amplifying-fsharp/sessions/issues/25
- Problem description

Please let us know If this works for you or suggest further improvements.

### CODE REVIEW

File: src/content/sessions/2023-09-01.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+title: "Rider: CE custom operations completion list"
+preview: ""
+isDraft: true
+date: 2023-09-01T13:00:00.000Z
+slug: "2023/09/01"
+champion: Jindřich Ivánek
+zoomLink: ""
+zoomPasscode: ""
+issueLink: "https://youtrack.jetbrains.com/issue/RIDER-15832/Put-custom-operations-at-the-top-of-completion-list-if-its-invoked-on-top-level-inside-computation-expression"
+company: ""
```
Comment (nojaf) [inline src/content/sessions/2023-09-01.md:11]: This can be `Ciklum` I think, see https://github.com/amplifying-fsharp/sessions/issues/25

Comment (edgarfgp) [reply src/content/sessions/2023-09-01.md]: Good spot. Thanks

File: src/content/sessions/2023-09-01.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Rider: CE custom operations completion list"
+preview: ""
+isDraft: true
+date: 2023-09-01T13:00:00.000Z
```
Comment (jindraivanek) [inline src/content/sessions/2023-09-01.md:5]: Unfortunately, 2023-09-01 is not possible for me. How about 2023-09-08 ? Any time until 16:00.

File: src/content/sessions/2023-09-01.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Currently it's next to impossible to see what custom operations are available:
+
+<img alt="Currently it's next to impossible to see what custom operations are available" class="img-fluid" src="/images/sessions/ceCustomOperationCompletion.png"/>
+
+The goal of the session is to improve the situation by putting custom operations at the top of the completion list if it's invoked inside computation expression.
+
+## Champions
+
+- [Jindřich Ivánek](https://twitter.com/jindraivanek)
```
Comment (jindraivanek) [inline src/content/sessions/2023-09-01.md:25]: Please change back to `https://github.com/jindraivanek`. I added link to socials and blog there. Thanks.

**Comment by jindraivanek** [comment]
Thanks, for creating this 🙂 I added comments as review.

File: src/content/sessions/2023-09-01.md (hunk 2 above)
Comment (edgarfgp) [reply src/content/sessions/2023-09-01.md:5]: Sure that works for us

File: src/content/sessions/2023-09-01.md (hunk 3 above)
Comment (edgarfgp) [reply src/content/sessions/2023-09-01.md]: Updated :)

**Pull request merged**